doc/source/release-notes/Percona-XtraDB-Cluster-8.0.26-16.1.rst

```diff
@@ -0,0 +1,51 @@
+.. _PXC-8.0.26-16:
+
+================================================================================
+*Percona XtraDB Cluster* 8.0.26-16
+================================================================================
+
+:Date: 2022
+:Installation: `Installing Percona XtraDB Cluster <https://www.percona.com/doc/percona-xtradb-cluster/8.0/install/index.html>`_
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and security features your business requires to meet your customer expectations and business goals.
+
+Release Highlights
+================================================================================
+
+The following are some of the notable fixes for MySQL 8.0.26, provided by Oracle, and included in this release:
+
+* The `TLSv1 and TLSv1.1 <https://tools.ietf.org/id/draft-ietf-tls-oldversions-deprecate-02.html>`__ connection protocols are deprecated.
+* Identifiers with specific terms, such as "master" or "slave" are deprecated and replaced. See the `Functionality Added or Changed section in the 8.0.26 Release Notes <https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-26.html#mysqld-8-0-26-feature>`__ for a list of updated identifiers. The following terms have been changed:
+    - The identifier ``master`` is changed to ``source``
+    - The identifier ``slave`` is changed to ``replica``
+    - The identifier ``multithreaded slave`` (``mts``) is changed to ``multithreaded applier`` (``mta``)
+* When using semisynchronous replication, either the old version or the new version of system variables and status variables are available. You cannot have both versions installed on an instance. Be aware, the new system variables are available when you use the new version, but the old values are not. The old system variables are available when you use the old version, but the new ones are not. The names are the following:
+    - For the source, the ``rpl_semi_sync_master`` plugin (``seminsync_master.so`` library) is the old version and the ``rpl_semi_sync_source`` plugin(``semisync_source.so`` library) is the new version.
```

Comment:
Hi, 
Could you also add information that rpl_semi_sync_source  and pl_semi_sync_replica plugins should be enabled after all the nodes of cluster are updated to the 8.0.26.  If the rpl_semi_sync_source/pl_semi_sync_replica plugin is enabled when not all the nodes are on  8.0.26 this will lead to data inconsistency between the nodes.

Reply:
Done.